### PR TITLE
fix keadm beta reset failed on edge node

### LINKED ISSUE #3870: run `keadm beta reset` fail
**What happened**:

```console
# keadm beta reset
Error: failed to create KubeClient, error: get kube config failed with error: stat /root/.kube/config: no such file or directory
Usage:
  keadm beta reset [flags]

Examples:

For cloud node:
keadm beta reset

For edge node:
keadm beta reset


Flags:
      --force                Reset the node without prompting for confirmation
  -h, --help                 help for reset
      --kube-config string   Use this key to set kube-config path, eg: $HOME/.kube/config (default "/root/.kube/config")

execute keadm command failed:  failed to create KubeClient, error: get kube config failed with error: stat /root/.kube/config: no such file or directory

```

**What you expected to happen**:

reset edge node success

**How to reproduce it (as minimally and precisely as possible)**:

run `keadm beta reset` on an edge node

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): ``
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`): `version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"3803951602f938d9d90d74957eb0fbc238142101", GitTreeState:"clean", BuildDate:"2022-03-14T02:30:42Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
`
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:
fix keadm beta reset failed on edge node
on edge node, we don't have /root/.kube/config file, so keadm exit. so just ignore the error
**Which issue(s) this PR fixes**:

Fixes #3870 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/beta/reset.go [hunk 1]
```diff
@@ -67,17 +67,16 @@ func NewKubeEdgeResetBeta() *cobra.Command {
...
-				IsEdgeNode = true
-			case common.NoneRunning:
+			whoRunning := util.RunningModuleV2(reset)
+			if whoRunning == common.NoneRunning {
 				fmt.Println("None of KubeEdge components are running in this host, exit")
 				os.Exit(0)
 			}
+
+			if whoRunning == common.KubeEdgeEdgeRunning {
+				IsEdgeNode = true
```
Comment (wackxu) [inline keadm/cmd/keadm/app/cmd/beta/reset.go:77]: s/IsEdgeNode/isEdgeNode/g

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 2]
```diff
@@ -149,24 +149,19 @@ func GetOSInterface() types.OSTypeInstaller {
...
-		return types.NoneRunning, err
-	}
+	cloudCoreRunning, _ := IsCloudcoreContainerRunning(constants.SystemNamespace, opt.Kubeconfig)
 	if cloudCoreRunning {
-		return types.KubeEdgeCloudRunning, nil
+		return types.KubeEdgeCloudRunning
 	}
 
-	edgeCoreRunning, err := osType.IsKubeEdgeProcessRunning(KubeEdgeBinaryName)
+	edgeCoreRunning, _ := osType.IsKubeEdgeProcessRunning(KubeEdgeBinaryName)
```
Comment (wackxu) [inline keadm/cmd/keadm/app/cmd/util/common.go:159]: print the err?

Comment (wackxu) [reply keadm/cmd/keadm/app/cmd/util/common.go:159]: if there is a real err when check cloudcore/edgecore  running,  ignore the err may have problem, we should recommend user to have retry

**Comment by gy95** [comment]
PTAL cc @wackxu @fisherxu

**Comment by gy95** [comment]
rebased, Please review again @wackxu @fisherxu

**Comment by wackxu** [comment]
lgtm

**Comment by fisherxu** [comment]
Let's pick the commit to release-1.11 @gy95

**Pull request merged**